src/test/java/io/redis/examples/QueryFtExample.java

```diff
@@ -0,0 +1,334 @@
+            + "that pressure. Rim brakes deliver satisfactory braking control, and the wide tires "
+            + "provide a smooth, stable ride on paved roads and gravel. Rack and fender mounts "
+            + "facilitate setting up the Roll Low-Entry as your preferred commuter, and the "
+            + "BMX-like handlebar offers space for mounting a flashlight, bell, or phone holder.\","
+            + "    \"condition\": \"new\""
+            + "  }",
+
+            "  {"
+            + "    \"pickup_zone\": \"POLYGON((1.9450 41.4301, 2.4018 41.4301, 2.4018 41.1987, "
+            + "1.9450 41.1987, 1.9450 41.4301))\","
+            + "    \"store_location\": \"2.1734, 41.3851\","
+            + "    \"brand\": \"nHill\","
+            + "    \"model\": \"Summit\","
+            + "    \"price\": 1200,"
+            + "    \"description\": \"This budget mountain bike from nHill performs well both "
+            + "on bike paths and on the trail. The fork with 100mm of travel absorbs rough "
+            + "terrain. Fat Kenda Booster tires give you grip in corners and on wet trails. "
+            + "The Shimano Tourney drivetrain offered enough gears for finding a comfortable "
+            + "pace to ride uphill, and the Tektro hydraulic disc brakes break smoothly. "
+            + "Whether you want an affordable bike that you can take to work, but also take "
+            + "trail in mountains on the weekends or you’re just after a stable, comfortable "
+            + "ride for the bike path, the Summit gives a good value for money.\","
+            + "    \"condition\": \"new\""
+            + "  }",
+
+            "  {"
+            + "    \"pickup_zone\": \"POLYGON((12.4464 42.1028, 12.5464 42.1028, "
+            + "12.5464 41.7028, 12.4464 41.7028, 12.4464 42.1028))\","
+            + "    \"store_location\": \"12.4964,41.9028\","
+            + "    \"model\": \"ThrillCycle\","
+            + "    \"brand\": \"BikeShind\","
+            + "    \"price\": 815,"
+            + "    \"description\": \"An artsy,  retro-inspired bicycle that’s as "
+            + "functional as it is pretty: The ThrillCycle steel frame offers a smooth ride. "
+            + "A 9-speed drivetrain has enough gears for coasting in the city, but we wouldn’t "
+            + "suggest taking it to the mountains. Fenders protect you from mud, and a rear "
+            + "basket lets you transport groceries, flowers and books. The ThrillCycle comes "
+            + "with a limited lifetime warranty, so this little guy will last you long "
+            + "past graduation.\","
+            + "    \"condition\": \"refurbished\""
+            + "  }"
+        };
+
+        for (int i = 0; i < bicycleJsons.length; i++) {
+            jedis.jsonSet("bicycle:" + i, new Path2("$"), bicycleJsons[i]);
+        }
+
+        // STEP_START ft1
+        SearchResult res1 = jedis.ftSearch(
+            "idx:bicycle",
+            new Query("@description: kids")
```

Comment:
For the time being, my preference is to use FTSearchParams instead of Query which may have some features missing.

Please consider this comment for all the PRs.

Reply:
@sazzad16  I've removed Query from this PR (no need for any search params in this one) and I'll update the others to use FTSearchParams.